### ISSUE TITLE
InvalidArgumentException: Start date must be anterior to end date.

### ISSUE THREAD

**Issue by jcqtc** [fzaninotto/Faker#1498]
Every new PR have this error during the Travis build:

```
1) Faker\Test\Provider\DateTimeTest::testFixedSeedWithMaximumTimestamp
InvalidArgumentException: Start date must be anterior to end date.

/home/travis/build/fzaninotto/Faker/src/Faker/Provider/DateTime.php:134
/home/travis/build/fzaninotto/Faker/src/Faker/Provider/DateTime.php:216
/home/travis/build/fzaninotto/Faker/test/Faker/Provider/DateTimeTest.php:251
```

I can't detail more as I really don't understand where this error come from...

Here's the class:

``` php
public static function dateTimeBetween($startDate = '-30 years', $endDate = 'now', $timezone = null)
    {
        $startTimestamp = $startDate instanceof \DateTime ? $startDate->getTimestamp() : strtotime($startDate);
        $endTimestamp = static::getMaxTimestamp($endDate);
        if ($startTimestamp > $endTimestamp) {
            throw new \InvalidArgumentException('Start date must be anterior to end date.');
        }
        $timestamp = mt_rand($startTimestamp, $endTimestamp);
        return static::setTimezone(
            new \DateTime('@' . $timestamp),
            $timezone
        );
    }
```

**Comment by ppelgrims** [comment]
Fixed by #1469

**Comment by jcqtc** [comment]
@ppelgrims cool!
